docs/workflow-controller-configmap.md

```diff
@@ -40,6 +40,47 @@ spec:
       nodeSelector:
               kubernetes.io/os: linux
 ```
+```yaml
```

Comment:
???

Reply:
It is an example similar to the workflow-controller one above it to show where you specify the configmap arg.